src/utils/helper.ts

```diff
@@ -0,0 +1,39 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export const createTodo = (title: string): Todo => {
+  return {
+    id: +new Date(),
```

Comment:
You can use [Crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)